UnityProject/Assets/Editor/LevelBuilder/GUI/LevelBuilderWindow.cs

```diff
@@ -19,16 +19,24 @@
 
 public class LevelBuilderWindow : EditorWindow
 {
+    public static float SectionTitleHeight = 20;
+
     private static LevelBuilderWindow windowInstance = null;
     private int optionChosenByUser = 0;
 
     // Main menus.
-    private IMenu[] menus =
+    private IMenu[] menus;
+
+    public void OnEnable()
     {
-        new TerrainMenu(),
-        new PickupMenu(),
-        new ObstacleMenu(),
-    };
+        menus = new IMenu[]
+        {
+            new TerrainMenu(),
+            new OverlayMenu(),
```

Comment:
bear in mind we'll have different overlays that might be toggable independently, like the compass not only the grid

Reply:
Yeah this is the menu for the section to put all of those toggles